rootio/wbuffer_test.go

```diff
@@ -579,9 +579,6 @@ func TestWBuffer_Write(t *testing.T) {
 func TestWriteWBuffer(t *testing.T) {
 	for _, test := range rwBufferCases {
 		t.Run("write-buffer="+test.file, func(t *testing.T) {
-			if test.skip {
```

Comment:
as this was the last test that had to be skipped, you can remove the `skip` field from the `rwBufferCases`.

Reply:
Done